test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -396,6 +396,31 @@
       initPods: 20000
       measurePods: 5000
 
+- name: PreemptionPV
```

Comment:
minor nit: `PreemptionPVs` to stay consistent with the other PV workloads

Reply:
Done